http_check/datadog_checks/http_check/data/conf.yaml.example

```diff
@@ -340,12 +340,12 @@ instances:
     #   Host: <ALTERNATIVE_HOSTNAME>
     #   X-Auth-Token: <AUTH_TOKEN>
 
-    ## @param host_header_ssl - boolean - optional - default: false
+    ## @param tls_use_host_header - boolean - optional - default: false
     ## If set to true, the Host header is used to match 
     ## against the SSL Certificate CN or SAN.
     ## Requires persist_connections set to true.
```

Comment:
We can probably also update the description to match the other ones: https://github.com/DataDog/integrations-core/pull/5833/files#diff-dff19a9bccd81f12d67265fc8a46b798R263-R267